tradeexecutor/visual/equity_curve.py

```diff
@@ -125,3 +126,13 @@ def calculate_aggregate_returns(equity_curve: pd.Series, freq: str = "BM"):
     return sampled.pct_change()
 
 
+def get_daily_returns(state: State) -> (pd.Series | None):
+    """Used for advanced statistics
+
+    :returns:
+        If valid state provided, returns are returned as calendar day (D) frequency, else None"""
+    if isinstance(state, State):
```

Comment:
None check here is no tneeded, because there is no case where state would be None - it simply is not optional in any part of the code.

Reply:
To remove after discussion with @miohtama